ciphers/autoclave.py

```diff
@@ -0,0 +1,111 @@
+def encrypt(plaintext, key) -> str:
```

Comment:
Clave is Latin for key so the more popular name for this algorithm is autokey.
```suggestion
”””
https://en.wikipedia.org/wiki/Autokey_cipher
> An autokey cipher (also known as the autoclave cipher) is a cipher that incorporates the message (the plaintext into the key. The key is generated from the message in some automated fashion, sometimes by selecting certain letters from the text or, more commonly, by adding a short primer key to the front of the message.
”””


def encrypt(plaintext, key) -> str:
```